### PR TITLE
Add more steps to the play with grafana mimir page

### LINKED ISSUE #7844: Docs feedback: /docs/sources/mimir/get-started/play-with-grafana-mimir/index.md
In doing the demo at https://grafana.com/docs/mimir/latest/get-started/play-with-grafana-mimir/,
my page at http://localhost:9000/alerting/new/alerting  lacks the two field below
I'm therefore blocked.

- Type example-namespace in the “Namespace” field.
- Type example-group in the “Group” field.

Screenshot_2024-04-08_09-44-04

Am I even using the right Grafana? This looks like one that failed for us before...

### PR DESCRIPTION
**Pull request by davecb**
I've fleshed out the steps slightly, as this is my (and others') first time through, and one doesn't often know what is needed.

#### What this PR does

Adds a clarification to instructions which can be unclear the first time one does it.
As this is a demonstration recommended to new adopters,  a lot of first-time users
will be trying it.

#### Which issue(s) this PR fixes or relates to

/docs/sources/mimir/get-started/play-with-grafana-mimir/index.md #7844 in which I
failed to guess the right option, and had to ask

Fixes #7844

#### Checklist

- [ ] Tests updated.
- [ x ] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

**Comment by davecb** [comment]
There are other issues in the "play with grafana" page.
For example, the section on doing a reporting rule is (nearly?) identical to the section on alert rules

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+F---
```
Comment (jhesketh) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:1]: This looks like a typo
```suggestion
---
```

**Review by jhesketh — COMMENTED** [review]
Thanks so much for the fixes :-).

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 2]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
 3. Configure the recording rule:
    1. Select `Mimir or Loki recording rule` in the top selector.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:121]:
```suggestion
   1. Select **Mimir or Loki recording rule** in the top selector.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 3]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
...
    2. Choose `Mimir` in the "Select data source" field.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:122]:
```suggestion
   1. Choose **Mimir** in the **Select data source** field.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 4]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
...
-   3. Type `sum(up)` in the "Metrics browser" query field.
-   4. Type `sum:up` in the "Rule name" field.
-   5. Type `example-namespace` in the "Namespace" field.
-   6. Type `example-group` in the "Group" field.
-   7. From the upper-right corner, click the **Save and exit** button.
+   3. Type `sum:up` in the "Rule name" field.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:123]:
```suggestion
   1. Type `sum:up` in the **Rule name** field.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 5]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
...
+   4. Choose "Code" in the "Builder | Code" field on the right.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:124]:
```suggestion
   1. Choose **Code** in the **Builder | Code** field on the right.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 6]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
...
+   5. Type `sum(up)` in the "Metrics browser" query field.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:125]:
```suggestion
   1. Type `sum(up)` in the **Metrics browser** query field.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 7]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
...
+   6. Choose  "Data source-managed" in the "Grafana-managed | Data source-managed" in the "Rule Type" paragraph.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:126]:
```suggestion
   1. Choose  **Data source-managed** in the **Grafana-managed | Data source-managed** field in the **Rule Type** paragraph.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 8]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
...
+   7. Type `example-namespace` in the "Namespace" field.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:127]:
```suggestion
   1. Type `example-namespace` in the **Namespace** field.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 9]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
...
+   8. Type `example-group` in the "Group" field.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:128]:
```suggestion
   1. Type `example-group` in the **Group** field.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 10]
```diff
@@ -120,11 +120,13 @@ offered by Grafana.
...
+   9. From the upper-right corner, click the **Save and exit** button.
```
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:129]:
```suggestion
   1. From the upper-right corner, click the **Save and exit** button.
```

**Review by jdbaldry — COMMENTED** [review]
Unrelated to your change but since we are here, it may be nice to get the style in-keeping with Writers' Toolkit guidelines.

- **bold** is used for UI elements: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold.

- We can make use of the Markdown renderer knowing how to auto-increment list numbers and just `1.` as the number for each step: https://grafana.com/docs/writers-toolkit/write/markdown-guide/#numbered-lists

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md (hunk 2 above)
Comment (davecb) [reply docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:121]: I was just doing copy-pasta, will apply your fixes

Comment (jdbaldry) [reply docs/sources/mimir/get-started/play-with-grafana-mimir/index.md]: Yeah, I completely understand :) thanks for considering the changes!

Apologies for the inconsistent style.

As an interim owner of the docs, I'm looking to clean bits up as I go and I appreciate any contributions that can help me do that.

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md [hunk 11]
```diff
@@ -118,13 +118,15 @@ offered by Grafana.
...
-   4. Type `sum:up` in the "Rule name" field.
-   5. Type `example-namespace` in the "Namespace" field.
-   6. Type `example-group` in the "Group" field.
-   7. From the upper-right corner, click the **Save and exit** button.
+   1. Select **Mimir or Loki recording rule** in the top selector.
+   1. Choose **Mimir** in the **Select data source** field.
+   1. Type `sum:up` in the **Rule name** field.
+   1. Choose **Code** in the **Builder | Code** field on the right.
+   1. Type `sum(up)` in the **Metrics browser** query field.
+   1. Choose  **Data source-managed** in the **Grafana-managed | Data source-managed** field in the **Rule Type** paragraph.
```
Comment (56quarters) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:126]:
```suggestion
   1. Choose **Data source-managed** in the **Grafana-managed | Data source-managed** field in the **Rule Type** paragraph.
```

**Pull request merged**